Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -350,6 +351,10 @@ def set(self, issue, assignee=None, opened=None, why=None, project=None, compone
                 issue._component = component
                 issue._version = version
 
+        if source_change:
+            sys.stderr.write('Bugzilla does not support source changes at this time\n')
+            return None
```

Comment:
Does this make sense to `return None` in this case? Seems like it could act more as a warning instead of somewhat of a "fatal" operation to this command by not returning the issue back

Reply:
My thinking here is that if we don't return `None`, a program wouldn't be able to tell that its request to `set` the `source_change` failed. It may be the case that program doesn't do anything with the success/failure state (and right now, no program is using this particular code path), but it seems like we should at least make it detectable? Especially in the case where the program is actually a service whose logs we rarely inspect.